src/topology/continuous_function/algebra.lean

```diff
@@ -582,6 +582,36 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [semiring R] [topological_space R]
+  [topological_ring R] : has_continuous_mul C(α, R) :=
+  ⟨begin
+    refine continuous_of_continuous_uncurry _ _,
+    change continuous ((λ (x : R × R), x.1 * x.2) ∘
+      (λ (x : (C(α, R) × C(α, R)) × α), (x.1.1 x.2, x.1.2 x.2))),
+    refine (continuous_fst.comp (continuous.prod_mk _ _)).mul
+      (continuous_snd.comp (continuous.prod_mk _ _)),
+    any_goals { change continuous ((λ (x : C(α, R) × α), x.1 x.2) ∘
+        (λ (x : (C(α, R) × C(α, R)) × α), (x.1.1, x.2))),
+      refine continuous.comp continuous_ev
+        ((continuous_fst.comp continuous_fst).prod_mk continuous_snd), },
```

Comment:
I don't love this `any_goals` style.  I understand you're doing it because the same goal turns up repeatedly.  But it would be better style to phrase it as a `have` statement, something like this:
```lean
    have h : continuous ((λ (x : C(α, R) × α), x.1 x.2) ∘
        (λ (x : (C(α, R) × C(α, R)) × α), (x.1.1, x.2))),
    { refine continuous.comp continuous_ev
        ((continuous_fst.comp continuous_fst).prod_mk continuous_snd) },
```
Same for the other `any_goals`.

Reply:
I have changed it, I can't use `convert` and `any_goals` together..